conftest.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
This file should not be necessary at all, `barril.fixtures` should be registered as an [entry-point](https://docs.pytest.org/en/latest/writing_plugins.html#making-your-plugin-installable-by-others) so it is discovered and loaded by pytest automatically.

```python
entry_points={"pytest11": ["barril = barril.fixtures"]},
```

Reply:
If the fixtures are public, that is.